apps/emqx/i18n/emqx_schema_i18n.conf

```diff
@@ -1541,7 +1489,8 @@ Enable TLS session reuse.
 common_ssl_opts_schema_depth {
     desc {
         en: """
-Maximum number of non-self-issued intermediate certificates that can follow the peer certificate in a valid certification path. So, if depth is 0 the PEER must be signed by the trusted ROOT-CA directly; if 1 the path can be PEER, CA, ROOT-CA; if 2 the path can be PEER, CA, CA, ROOT-CA, and so on. The default value is 10.
+Maximum number of non-self-issued intermediate certificates that can follow the peer certificate in a valid certification path.
+So, if depth is 0 the PEER must be signed by the trusted ROOT-CA directly; if 1 the path can be PEER, CA, ROOT-CA; if 2 the path can be PEER, CA, CA, ROOT-CA, and so on. The default value is 10.
```

Comment:
for 
>  if 1 the path can be PEER, CA, ROOT-CA;

maybe replace `CA` with `intermediate CA` ?

Reply:
will fix in the next PR.